CONTRIBUTING.md

```diff
@@ -22,12 +22,23 @@ And run the test suite (should be all green with pending tests):
 
 If you have problems installing nmatrix, please consult the [nmatrix installation wiki](https://github.com/SciRuby/nmatrix/wiki/Installation) or the [mailing list](https://groups.google.com/forum/#!forum/sciruby-dev).
 
+**NOTE**: `Daru` is compatible with Ruby versions < 2.5; for later Ruby versions it breaks, returning the following error when `bundle exec rspec` is run for Ruby 
+versions >= 2.5.
+
+```
+bundler: failed to load command: rspec (/home/.rbenv/versions/2.5.1/bin/rspec)
+...: command not found
+```
+To reproduce this issue or explore this error further, head over to [issue #500](https://github.com/SciRuby/daru/issues/500),
```

Comment:
Please add this as well : 

'Also, if you want to fix this issue, then we can discuss it here : https://github.com/SciRuby/daru/issues/505'